src/util/tr.js

```diff
@@ -466,6 +469,52 @@ const _tr = (msg, {
       }
     } else {
       renderFooterNotification(targetingMsgJson)
+
+      if (window.clevertap.hasOwnProperty('popupCallback') &&
+        typeof window.clevertap.popupCallback !== 'undefined' &&
+        typeof window.clevertap.popupCallback === 'function') {
+        const popupCallback = window.clevertap.popupCallback
+
+        const inaObj = {}
+        inaObj.msgContent = targetingMsgJson.msgContent
+        inaObj.msgId = targetingMsgJson.wzrk_id
+
+        var msgCTkv = []
+        for (var wzrkPrefixKey in targetingMsgJson) {
+          // ADD WZRK PREFIX KEY VALUE PAIRS
+          if (wzrkPrefixKey.startsWith('wzrk') && wzrkPrefixKey !== 'wzrk_id') {
```

Comment:
Let's move prefix keys in the constants class. Reference [here](https://github.com/CleverTap/clevertap-ios-sdk/blob/master/CleverTapSDK/CTConstants.h#L62).